web_external/models/FeaturesetModel.js

```diff
@@ -10,5 +10,13 @@ isic.models.FeaturesetModel = isic.Model.extend({
 
     creator: function () {
         return new isic.models.UserModel(this.get('creator'));
+    },
+
+    destroy: function (options) {
+        var params = _.clone(options);
```

Comment:
It's good practice for functions not to mutate their parameters unless documented. I want `wait` to always be `true`, regardless of what is passed in. Since `_.extend` will mutate it's first argument, `_.clone` seems like the best way to go.

I'll comment though. Let me know if you see a cleaner way to do this.